### DISCUSSION TITLE
Can't pull :latest image on Raspberry PI 4

### DISCUSSION THREAD

**Discussion by Gollair** [photoprism/photoprism#1813] [Q&A] [answered]
Hi all, 

have photoprism up an running with the photoprism-arm64:latest image. Have now changed the image in the yml to photoprism:latest to get updates. If I now want to compose the docker I get the following error message:

ERROR: no matching manifest for linux/arm/v7 in the manifest list entries

Any ideas? Thanks.

**Comment by lastzero** [comment, marked as answer]
See https://docs.photoprism.org/getting-started/raspberry-pi/:
> Our stable version and development preview have been built into a single multi-arch image for 64-bit AMD, Intel, and ARM processors. That means, Raspberry Pi 3 / 4 owners can pull from the same repository

Stable Release: `photoprism/photoprism:latest`
Development Preview: `photoprism/photoprism:preview`

**Reply by lastzero** [reply]
Also you may need to enable 64-bit mode on your Pi. It's also described in our docs. "linux/arm/v7" is the wrong image for a RP4.

**Reply by Gollair** [reply]
Thanks for your quick reply. Tried both images (latest and preview), but get the error message mentioned above for both.

**Reply by Gollair** [reply]
64-bit mode is enabled.

**Reply by lastzero** [reply]
> To ensure compatibility with 64-bit Docker images, your Raspberry Pi 3 / 4 must boot with the arm_64bit=1 flag in its config.txt file. An "exec format error" will occur otherwise.

Will add your error message to the list...

**Reply by lastzero** [reply]
Then something is wrong, ARMv7 is 32-bit. Maybe an old operating system? The ARMv7 build is still failing due to an issue with Docker or Go. You can use an old multi-arch build like `photoprism/photoprism:20211203` from Docker Hub though: https://hub.docker.com/r/photoprism/photoprism/tags

**Reply by lastzero** [reply]
Docs have been updated: https://docs.photoprism.org/getting-started/raspberry-pi/#older-armv7-based-devices

**Reply by Gollair** [reply]
Thank you. That was very helpful.

**Reply by lastzero** [reply]
photoprism/photoprism:armv7 has been pushed to Docker Hub. Does it work for you? Includes all our latest improvements.

**Reply by Gollair** [reply]
Works pretty well, thanks for that.

**Reply by lastzero** [reply]
Thanks for testing! 🤗

**Reply by lastzero** [reply]
see https://github.com/photoprism/photoprism/issues/1815#issuecomment-994773932

**Reply by lastzero** [reply]
Added a Raspbian section to our docs, maybe this helps in your case as well: https://docs.photoprism.app/getting-started/raspberry-pi/#no-matching-manifest

**Reply by Gollair** [reply]
Thank you, pull works perfectly
